docs/api/fab.md

```diff
@@ -41,6 +41,26 @@ import Positioning from '@site/static/usage/v7/fab/positioning/index.md';
 
 <Positioning />
 
+### Safe Area
+
+If there is no `ion-header` or `ion-footer` component, the fab may be covered by a device's notch, status bar, or other device UI. In these cases, the [safe area](/docs/layout/safe-area) on the top and bottom is not taken into account. This can be adjusted by using the [`--ion-safe-area-(dir)` variables](#application-variables).
```

Comment:
The `safe area` link brings me to a `Page Not Found` and `#application-variables` doesn't seem to exist?

Reply:
Thanks for catching those! Fixed in 5b13f717e873e9d98ac5f0ebbffe31ae29b1012d